src/test/java/de/caritas/cob/userservice/api/service/session/SessionServiceTest.java

```diff
@@ -590,6 +592,48 @@ void fetchSessionForConsultant_Should_Return_ValidConsultantSessionDTO() {
     assertEquals(session.getGroupId(), result.getGroupId());
     assertEquals(session.getFeedbackGroupId(), result.getFeedbackGroupId());
     assertEquals(session.getConsultingTypeId(), result.getConsultingType().intValue());
+    assertEquals(session.getUserAge(), result.getAge());
```

Comment:
if it's not too much effort, could you please change this test to use assertJ assertThat fluent assertions instead?

Reply:
hmm no no much, but I don't like that mix :/ also in the same class. So I think a bit more, that you should keep consitency in the place around.

Reply:
agreed, it should be changed for the whole class or kept consitent. So if changing for the whole class is not an option for the moment, it can stay as it is

Reply:
I agrree